hardware/OpenCustomizationSelector/src/com/sonymobile/customizationselector/NS/NSBootReceiver.java

```diff
@@ -1,27 +1,27 @@
 package com.sonymobile.customizationselector.NS;
 
-import android.content.BroadcastReceiver;
 import android.content.Context;
 import android.content.Intent;
 import android.os.UserHandle;
 import android.provider.Settings;
 import com.sonymobile.customizationselector.CSLog;
 import com.sonymobile.customizationselector.CommonUtil;
 
-public class NSBootReceiver extends BroadcastReceiver {
+public class NSBootReceiver extends NSBootReceiver {
+    private static final String TAG = "NSBootReceiver";
 
     @Override
     public void onReceive(Context context, Intent intent) {
         if (context == null) {
-            CSLog.d("NSBootReceiver", "Error: Context was null");
+            CSLog.d(TAG, "Error: Context was null");
```

Comment:
Shouldn't this be `CSLog.e`, for `Error`?
It is currently `CSLog.d`, for `Debug`, but is printing `Error: Context was null`.

Reply:
Makes sense, resolved in 6424d1d6 unifying also similar things